newIDE/app/public/external/piskel/piskel-main.js

```diff
@@ -26,15 +26,19 @@ const closeWindow = () => {
 
 // Repeatedly try to gain access to piskel's control element and its methods
 // When succeeding, stop trying.
+const editorFrameEl = document.getElementById('piskel-frame');
+editorFrameEl.src = 'piskel-editor/index.html';
```

Comment:
To be extra cautious, I think we should set the ".src" AFTER registering the onload (to be sure that document is not loaded before the onload is attached - very unlikely but we never now).

Reply:
ah yes, you're right. Updating now